public/index.php

```diff
@@ -10,9 +10,11 @@
 chdir(dirname(__DIR__));
 require 'vendor/autoload.php';
 
-/** @var \Interop\Container\ContainerInterface $container */
-$container = require 'config/container.php';
+call_user_func(function () {
```

Comment:
Please add a comment above this indicating the purpose. While I understand it, many do not, and it could be a useful educational exercise for those who look in the file.